python/lsst/verify/bin/print_metricvalues.py

```diff
@@ -21,6 +21,8 @@
 
 """Summarize measured metric values in one butler repo, or difference values
 between two repos.
+
+This tool is built on the `~lsst.verify.extract_metricvalues` module.
```

Comment:
Do you want to explicitly say that the module is available for anyone trying to use if from a non-command-line environment?